### ISSUE TITLE
Document required ordering of clauses for @use and @forward

### ISSUE THREAD

**Issue by bjg222** [sass/sass-site#568]
Documentation of the @use and @forward at-rules describes how to use the various clauses for namespacing (as), visibility (show/hide), and configuration (with) individually.  However, I would propose adding notes about the allowable ordering of these clauses.  The error dart-sass gives when compiling with clauses out of order can be vague if you're not aware of the ordering requirements, simply saying that a ';' was expected or similar.  It took some searching and testing to determine that "as", "show", and "hide" must come before "with".  It would be good to include this information in the online documentation.

**Comment by jathak** [comment]
Yeah, that makes sense. I've also filed https://github.com/sass/dart-sass/issues/1443 to have Dart Sass output better error messages for cases like these.
